jupyter/Scale Output Example.ipynb

```diff
@@ -0,0 +1,182 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Simple Scalar Output Example\n",
+    "\n",
+    "In this example, we will define an integer parameter called `Integer input` and will output a floating point called `My scalar output` that can be bound to a numeric dashboard tile."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scrapbook as sb \n",
+    "from random import *"
```

Comment:
`import *` is often discouraged in favor of either importing the module directly, or importing what you need from the module, like `import random` followed by `random.uniform()` or `from random import uniform` followed by `uniform()`. https://ni.github.io/python-styleguide/#O-1-6

Reply:
I'd also recommend moving the `random` import above the `scrapbook` import and adding a blank line between the two, since `random` is a standard library import which is typically lifted above 3rd party imports. https://ni.github.io/python-styleguide/#O-1-3